setup.py

```diff
@@ -22,11 +22,17 @@
 setuptools.setup(
     name="streamlit-bokeh",
     version="3.6.2",
-    author="Streamlit",
-    author_email="streamlitcommunity@snowflake.com",
+    author="Snowflake Inc",
+    author_email="hello@streamlit.io",
     description="Streamlit component that allows you to render Bokeh charts",
     long_description=long_description,
     long_description_content_type="text/markdown",
+    project_urls={
+        "Source Code": "https://github.com/streamlit/streamlit-bokeh",
+        "Bug Tracker": "https://github.com/streamlit/streamlit/issues",
```

Comment:
```suggestion
        "Bug Tracker": "https://github.com/streamlit/streamlit-bokeh/issues",
```

If by chance you really did intend to encourage issues go to the main library, I think we need to update the link in the README file, too.